### PR TITLE
switch Explore > Teams to new teams

### PR DESCRIPTION
**Pull request by whit537**
Rough but serviceable.

### CODE REVIEW

**Comment by whit537** [comment]
Ping @rohitpaulk.

**Comment by whit537** [comment]
I want this so I can see the teams we have on the site and what their profile pages look like as I think about #3456.

**Comment by whit537** [comment]
Sorry, failing test ...

**Comment by whit537** [comment]
Tests fixed, @rohitpaulk.

File: www/explore/teams/index.spt [hunk 1]
```diff
@@ -27,18 +18,18 @@ subhead = _("Teams")
...
     {% for team in teams %}
+    {% if team.is_approved or user.ADMIN %}
     <tr>
-        <td class=""><a href="/{{ team.name }}/">{{ team.name }}</a></td>
-        <td class="figure">{{ format_decimal(team.receiving, '#,##0.00') }}</td>
-        <td class="figure">{{ team.nmembers }}</td>
-        <td class="figure">{{ format_decimal(team.dollars_per_member, '#,##0.00') }}</td>
+        <td><a href="/{{ team.slug }}/">{{ team.name }}</a></td>
+        <td><a href="/{{ team.owner }}/">{{ team.owner }}</a></td>
+        <td>{{ team.status }}</td>
```
Comment (rohitpaulk) [inline www/explore/teams/index.spt]: Isn't the status column useless if all the teams shown are `approved`? Shouldn't this be shown only to admins?

Comment (whit537) [inline www/explore/teams/index.spt]: Good call. Fixed in cbf6cacf1e1785deb07d5893d59529e6d78780e8.

**Pull request merged**